osd/default_scc_modification.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Revert default Security Context Contraints modifications",
```

Comment:
Suggest: "Action required: Revert modifications to default Security Context Constraints"